packages/template-parser/src/index.ts

```diff
@@ -99,6 +100,9 @@ function preprocessNode(node: Node) {
 
   for (i = 0; i < keys.length; ++i) {
     const child = node[keys[i]];
+    if (child == null) {
```

Comment:
Without this defer block parsing fails with:
```
● parseForESLint() › @defer › should support the different variants of @defer

    TypeError: Cannot read properties of null (reading 'type')

      105 |     }
      106 |     const isArr = Array.isArray(child);
    > 107 |     if (child.type !== undefined) {
          |               ^
      108 |       // Angular sometimes uses a prop called type already
      109 |       child.__originalType = child.type;
      110 |     }

      at preprocessNode (src/index.ts:107:15)
      at preprocessNode (src/index.ts:129:11)
      at parseForESLint (src/index.ts:272:3)
      at Object.<anonymous> (tests/index.test.ts:559:23)
```

Maybe I got something wrong in the visitor keys?